SkillChat.Server/NETVersionComparison.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+
+namespace SkillChat.Server
+{
+    public class NETVersionComparison
+    {
+        /// <summary>
+        /// Получение наиболее подходящего Runtime .NET
+        /// </summary>
+        public static Runtime GetBestDotNETRuntime(string cfgVersion)
```

Comment:
Вместо cfgVersion лучше назвать targetVersion - целевая версия
```suggestion
        public static Runtime GetBestDotNETRuntime(string targetVersion)
```